### PR TITLE
[image] fix: blurry image on Safari when reloading the page

### PR DESCRIPTION
**Pull request by maimonav**
When reloading page with images in Safari, the `onLoad` event in `` is problematic, it fires only one time & due to inconsistency in Safari, sometimes it fires the `onLoad` without attaching the event to it, making the `onLoad` useless.
In order to fix that, let's fire the event, "again", when we certain that the image has been loaded.

### CODE REVIEW

**Review by shaulgo — APPROVED** [review]

File: packages/plugin-image/web/src/image-viewer.jsx [hunk 1]
```diff
@@ -275,7 +291,7 @@ class ImageViewer extends React.Component {
     const itemClassName = classNames(this.styles.imageContainer, className, {
       [this.styles.pointer]: hasExpand,
     });
-    const imageClassName = classNames(this.styles.image);
+    const imageClassName = this.styles.image;
```
[the remark this answers is not in the document]
Comment (maimonav) [reply packages/plugin-image/web/src/image-viewer.jsx:294]: :)

File: packages/plugin-image/web/src/image-viewer.jsx [hunk 2]
```diff
@@ -128,18 +143,19 @@ class ImageViewer extends React.Component {
     return (
       <img
         {...props}
+        id={fadeIn ? 'highRes' : undefined}
```
[the remark this answers is not in the document]
Comment (maimonav) [reply packages/plugin-image/web/src/image-viewer.jsx]: Sure.

File: packages/plugin-image/web/src/image-viewer.jsx [hunk 3]
```diff
@@ -18,16 +18,32 @@ import styles from '../statics/styles/image-viewer.scss';
...
+      const interval = setInterval(() => {
+        if (this.imageRef?.current?.complete) {
+          this.onImageLoad(this.imageRef.current);
+          clearInterval(interval);
+        }
+        if (++executionTimes === 3) {
+          clearInterval(interval);
+        }
+      }, 100);
+    }
```
Comment (shaulgo) [inline packages/plugin-image/web/src/image-viewer.jsx:46]: 1. extract function
2. Add to explanation that image's onLoad isn't always called on safari when refreshing there for... (if someone needs to do any changes in the future, he will need to know)

**Review by shaulgo — CHANGES_REQUESTED** [review]

File: packages/plugin-image/web/src/image-viewer.jsx (hunk 3 above)
Comment (maimonav) [reply packages/plugin-image/web/src/image-viewer.jsx]: done.

**Pull request merged**